internal/plugins/golang/init.go

```diff
@@ -51,3 +59,83 @@ func (p *initPlugin) Run() error {
 
 	return nil
 }
+
+const (
+	makefileBundleImgVarFragment = `# Bundle image URL
+BUNDLE_IMG ?= controller-bundle:latest`
+
+	makefileManifestsName = "manifests"
+	//nolint:lll
+	makefileManifestsFragment = `manifests: controller-gen
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	operator-sdk bundle generate -q --kustomize
+`
+
+	makefileBundleName     = "bundle"
+	makefileBundleFragment = `# Generate a bundle directory
+bundle: manifests
+	kustomize build config/olm-catalog | operator-sdk bundle generate -q --manifests
+`
+
+	makefileBuildBundleName     = "bundle-build"
+	makefileBuildBundleFragment = `# Build the bundle OCI image
+bundle-build: manifests
+	kustomize build config/olm-catalog | operator-sdk bundle generate -q --manifests --metadata --overwrite
+	operator-sdk bundle validate config/olm-catalog
+	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+`
+)
+
+func initUpdateMakefile() error {
+	makefileBytes, err := ioutil.ReadFile("Makefile")
+	if err != nil {
+		return err
+	}
+
+	makefileBytes = append([]byte(makefileBundleImgVarFragment), makefileBytes...)
+
+	// Modify Makefile with OLM recipes.
+	namedFragments := map[string]string{
+		makefileManifestsName:   makefileManifestsFragment,
+		makefileBundleName:      makefileBundleFragment,
+		makefileBuildBundleName: makefileBuildBundleFragment,
+	}
+	for name, fragment := range namedFragments {
+		makefileBytes = replaceOrAppendMakefileRecipe(makefileBytes, name, fragment)
+	}
+
+	return ioutil.WriteFile("Makefile", makefileBytes, 0644)
+}
+
+func replaceOrAppendMakefileRecipe(oldMakefile []byte, recipeName, fragment string) (newMakefile []byte) {
```

Comment:
You are customizing here the Makefile gen by the plugin. Am I right? 
So, it may be generic (customizeBullerplate) and added to kbutils since could be reused by others until we have the second phase. WDYT?

Reply:
Yes that's what's happening here. Probably a good idea to generalize this. I'm looking around for a makefile parser first because this update function isn't the most robust.

Reply:
I do not think that we should update the current targets. I mean, we can just add at the end of the file the new bundle targets.